src/views/home/PlayerHome/PlayerHome.vue

```diff
@@ -122,38 +175,84 @@
     get nextPuzzleID() {
       return this.eternaEssentialsAchievements[0].current_puzzle;
     }
+
+    data() {
+      return {
+        calendarOptions: {
+          initialView: 'upcoming',
+          headerToolbar: false,
+          plugins: [
+            dayGridPlugin,
+            interactionPlugin,
+            googleCalendarPlugin,
+            listPlugin,
+          ],
+          views: {
+            upcoming: {
+              type: 'list',
+              duration: { days: 3 },
+              listDayAltFormat: 'dddd',
+            },
+          },
+          height: 'auto',
+          eventClick: this.handleEventClick,
+          googleCalendarApiKey: process.env.VUE_APP_GOOGLE_API_ID,
+          events: {
+            googleCalendarId: process.env.VUE_APP_GOOGLE_CALENDAR_ID,
+          },
+        },
+      };
+    }
+
+    handleEventClick(info: {
+      jsEvent: { preventDefault: () => void };
+      event: { url: string | URL | undefined };
+    }) {
+      info.jsEvent.preventDefault(); // don't let the browser navigate
+
+      if (info.event.url) {
+        window.open(info.event.url);
+      }
+    }
   }
 </script>
 
 <style lang="scss" scoped>
-  @import '@/styles/global.scss';
```

Comment:
Not a big deal (again, lack of documented standards), but removing the indent on the CSS rules breaks my VSCode code folding. The template and script tags collapse, but the style tag doesn't. This should be something we handle with a linter/prettifier.